airbyte/caches/_catalog_backend.py

```
@@ -24,10 +23,7 @@
 )
 
 from airbyte.shared.catalog_providers import CatalogProvider
-
-
-if TYPE_CHECKING:
-    from sqlalchemy.engine import Engine
+from airbyte.shared.sql_processor import SqlConfig
```

Ruff (0.11.9)

Move application import `airbyte.shared.sql_processor.SqlConfig` into a type-checking block

Move into type-checking block

(TC001)